packages/desktop/views/onboarding/views/stronghold-setup/views/SetupStrongholdPasswordView.svelte

```diff
@@ -67,7 +68,12 @@
                     }
                 }
 
-                updateOnboardingProfile({ strongholdPassword })
+                updateOnboardingProfile({
```

Comment:
does this line cover all cases software profile cases?
I think if you import from stronghold it doesnt pass over this view